tests/plugins/test_koji_upload.py

```diff
@@ -1128,10 +1142,20 @@ def test_koji_upload_logs(self, tmpdir, monkeypatch, os_env, is_scratch, logs_re
             assert images[0].endswith(platform + ".tar.xz")
 
     @pytest.mark.parametrize('multiple', [False, True])
-    def test_koji_upload_multiple_digests(self, tmpdir, os_env,
-                                          multiple, reactor_config_map):
+    @pytest.mark.parametrize('manifest_available_with_digest', [False, True])
```

Comment:
Is there a test case for the time-out error that I'm missing?

Reply:
I removed the code snippet where we were capturing and re-raising it from the `util` module (we were not dealing with it at all).

In this case, we will let RegistrySession raise the timeout after its retry logic since we will not be able to determine whether the resources are available through the digest or not.

Reply:
`check_digest_availibility` still captures an exception and raises it under certain conditions.